primedev/core/filesystem/rpakfilesystem.cpp

```diff
@@ -342,15 +342,15 @@ void PakLoadManager::UnloadDependentPaks(PakHandle handle)
 static void HandlePakAliases(std::string& originalPath)
 {
 	// convert the pak being loaded to its aliased one, e.g. aliasing mp_hub_timeshift => sp_hub_timeshift
-	for (int64_t i = g_pModManager->m_LoadedMods.size() - 1; i > PakHandle::INVALID; i--)
+	for (int64_t i = g_pModManager->m_LoadedMods.size() - 1; i > -1; i--)
```

Comment:
Why removing `PakHandle::INVALID` while it already equals `-1`?

Reply:
We arent iterating over Pak handles here, so it was a mistake to use the named constant in the first place